addons/mail/static/src/models/activity_menu_view.js

```diff
@@ -18,6 +21,18 @@ registerModel({
     recordMethods: {
         close() {
             this.update({ isOpen: false });
+        },
+        async fetchData() {
+            const data = await this.messaging.rpc({
+                model: 'res.users',
+                method: 'systray_get_activities',
+                args: [],
+                kwargs: { context: session.user_context },
+            });
+            this.update({
+                activityGroups: insertAndReplace(data.map(vals => this.messaging.models['ActivityGroup'].convertData(vals))),
+                extraCount: 0,
+            });
             this.widget.renderElement();
```

Comment:
`close()` should still do `this.widget.renderElement()`

Reply:
Probably but as this is an intermediate commit it's probably not worth changing it, if the PR gets squashed the final code is correct I believe.

Reply:
No worry, this fix is cancelled by using component in the last commit, so it doesn't matter